buildconfig/CMake/CppCheck_Suppressions.txt.in

```diff
@@ -342,7 +342,7 @@ constVariable:${CMAKE_SOURCE_DIR}/qt/widgets/instrumentview/src/BankTextureBuild
 shadowFunction:${CMAKE_SOURCE_DIR}/qt/widgets/instrumentview/src/MiniPlotMpl.cpp:154
 shadowFunction:${CMAKE_SOURCE_DIR}/qt/widgets/instrumentview/src/MiniPlotMpl.cpp:164
 shadowFunction:${CMAKE_SOURCE_DIR}/qt/widgets/instrumentview/src/MiniPlotMpl.cpp:325
-unusedScopedObject:${CMAKE_SOURCE_DIR}/qt/widgets/plotting/src/ExternalPlotter.cpp:46
+unusedScopedObject:${CMAKE_SOURCE_DIR}/qt/widgets/plotting/src/ExternalPlotter.cpp:47
```

Comment:
Sorry, I think my explanation was I bit poor - what I meant was that we can fix the problem that this suppression refers to, and then we can remove this suppression line from this file.

i.e. I think the problem is that line 47 in `ExternalPlotter` is missing a `throw` in front of the `std::runtime_error`. Instead of throwing the runtime error, it is just creating a runtime error object and not doing anything with it, hence the warning "unusedScopedObject".

Reply:
Ahh, ok. Sorry I missed your point. I will try to fix it.